app/utils/lib/getURL.js

```diff
@@ -54,7 +55,7 @@ export const _getURL = (path, { cdn, full, cloud, cloud_route, cloud_params, _cd
 	}
 
 	if (full) {
-		return s.rtrim(siteUrl, pathPrefix) + url;
+		return siteUrl.replace(new RegExp(`${ escapeRegExp(/pathPrefix$/) }`, '')) + url;
```

Comment:
that's not how `escapeRegExp` works.. what you have to pass to it is the string you want to make sure doesn't behave like a regex, so you can use it's return on your own regex.. looks like the param for the `replace` is misplaced.. I think this fixes both issues:
```suggestion
		return siteUrl.replace(new RegExp(`${ escapeRegExp(pathPrefix) }$`), '') + url;
```

Reply:
I tried this approach and it worked :smiley ..  it's fixed, thanks